<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QDialog" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/chat/image/icon/icon_title1.png</normaloff>:/chat/image/icon/icon_title1.png</iconset>
  </property>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
   </property>
   <property name="currentText">
    <string>宋体</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fontSizeComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>541</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
</string>
   </property>
   <property name="currentText">
    <string>9</string>
   </property>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="textcolor">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_color.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>730</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_pic.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="textbold">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_bold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="textitalic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_italic.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WatermelonChat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/title.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emojiButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_emoji.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="clear">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_del.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendfile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_file.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>451</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 150);
border-radius: 20px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_download.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="textUnderline">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_underline.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gameButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>740</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_game.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="textButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditWrite">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 150);
border-radius: 20px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>730</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_voice.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hideButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>25</width>
     <height>6</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_hide.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/background_peopleleft.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
